LICENSE

```diff
@@ -178,15 +179,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
```

Comment:
this looks like a slightly derived license.  We should take a look at not deriving the license if possible.

Reply:
The LICENSE file that is in istio/common-files (which is being copied here) is the byte-for-byte copy of the original text from https://www.apache.org/licenses/LICENSE-2.0.txt, with the exception to the updated copyright date and copyright owner name on line 189.